phone_calls/nodejs/test.js

```diff
@@ -77,10 +77,10 @@ describe("Migration of data into TypeDB", function() {
 
 describe("Queries on phone_calls_nodejs database", function() {
     it("tests queries.js", async function() {
-        queries.processSelection(0, databaseName);
-
         await csvMigration.init(dataPath, databaseName);
 
+        queries.processSelection(0, databaseName);
+
```

Comment:
`processSelection` after the database is populated, which seems to fix the issues where the queries in the test failed because they were performed on an empty database.